jest.preset.js

```diff
@@ -0,0 +1,12 @@
+// @ts-check
+
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+
+const nxPreset = require('@nrwl/jest/preset');
```

Comment:
could we add a typings type for this import?
That way the `@ts-check` will work for us. Otherwise this will just be `any`!